src/utils/bootstrap/bootstrap.service.ts

```diff
@@ -189,91 +195,103 @@ export class BootstrapService {
         }
         user = await this.userService.getUserWithGroups(userInput.email);
 
-        if (!user) throw new Error('Unable to create group profiles');
+        if (!user)
+          throw new EntityNotInitializedException(
+            'Unable to create group profiles. User could not',
```

Comment:
Message looks odd?

Reply:
Fixed in 278fd646b688128804c35e41ef428f45ae9380d5